dlgr/griduniverse/experiment.py

```diff
@@ -1556,16 +1599,16 @@ def game_loop(self):
             self.socket_session.commit()
             count += 1
             self.grid.walls_updated = False
-            self.grid.food_updated = False
+            self.grid.objects_updated = False
             gevent.sleep(0.010)
 
             # TODO: Most of this code belongs in Gridworld; we're just looking
             # at properties of that class and then telling it to do things based
             # on the values.
 
-            # Log food updates every hundred rounds to capture maturity changes
-            if self.grid.food_maturation_threshold and (count % 100) == 0:
-                self.grid.food_updated = True
+            # Log object updates every hundred rounds to capture maturity changes
+            if self.grid.maturation_threshold and (count % 100) == 0:
```

Comment:
I think `self.grid.maturation_threshold` needs to be computed at launch using something like `any([o.get(`maturation_threshold`, 0) for o in self.object_config])`. That way this step will happen if any object type has a `maturation_threshold` set.